README.md

```diff
@@ -50,35 +50,69 @@ credentials as well as utility methods to create them and to get Application Def
 
 ## Quickstart
 
+### Using Maven
+
+#### Google Auth Library Bill of Materials
+In order to ensure transitive dependencies and the modules themselves are aligned with each other,
+we rely on the Google Auth Library Bill of Materials. Please add this to your dependency management
+section as follows:
+
+[//]: # ({x-version-update-start:google-auth-library-bom:released})
+```xml
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-bom</artifactId>
+      <version>1.30.1</version>
```

Comment:
Can this version be updated by release-please?

Reply:
Oh, I see the tag above it (`# ({x-version-update-start:google-auth-library-bom:released})`). If this is already auto updated, feel free to resolve this comment.